hypha/apply/funds/models/applications.py

```diff
@@ -343,6 +343,15 @@ def get_form_parameters(self, submission_id=None):
 
         return form_parameters
 
+    def get_form_fields(self, draft=False):
+        form_fields = super(RoundBase, self).get_form_fields(draft)
+        applicationform = self.fund.specific.forms.first()
+        if applicationform.minimize_form:
+            for field in form_fields.values():
+                if not field.required:
```

Comment:
It seems not all fields have the "required" attribute. I get a "'BlockFieldWrapper' object has no attribute 'required'" error when I test this locally. Wrapping this in a try/except seems to make it work.

Reply:
I hope this is not big problems, OTF and Reset would really like this feature I believe.

Reply:
Oh! I have had fixed this problem in my local. Forget to push it. Sorry.

Reply:
Fixed it.